app/Notifications/RequestVoucherForApprovedNotification.php

```
@@ -3,19 +3,18 @@
-use App\Models\DisbursementRequest;
+use App\Models\Voucher;
 use Illuminate\Bus\Queueable;
 use Notification;
 
-class RequestDisbursementVoucherForDeniedNotification extends Notification
+class RequestVoucherForApprovedNotification extends Notification
 {
     use Queueable;
     private $token;
     private $model;
     public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)